seed/static/seed/partials/update_item_labels_modal.html

```diff
@@ -3,7 +3,7 @@
     <h4 class="modal-title" id="manageLabelsModalLabel" translate>Add/Remove Labels</h4>
   </div>
   <div class="modal-body">
-    <div class="newLabelInput" ng-if="is_ali_root" style="margin-top: 0">
+    <div class="newLabelInput" ng-if="is_ali_root" style="margin-top: 0" ng-if="is_ali_root">
```

Comment:
looks like `ng-if="is_ali_root"` is already part of the "newLabelInput"

Reply:
ah! sorry i missed that. i'll clean it up